src/Aquifer.Public.API/Endpoints/Resources/Get/AvailableLanguages/Response.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Aquifer.Public.API.Endpoints.Resources.Get.AvailableLanguages;
+
+public class Response
+{
+    public int ContentId { get; set; }
+    public string ContentDisplayName { get; set; } = null!;
+    public int LanguageId { get; set; }
```

Comment:
Do we need language id here? I can't remember if we assign any relevance to that on the public API.

Reply:
You can query the search endpoint with it.